### DISCUSSION TITLE
After the update, an error appeared: Class "app\Settings\TGBotSettings" not found

### DISCUSSION THREAD

**Discussion by enshtein** [spatie/laravel-settings#202] [Q&A]
**[2023-02-20 16:08:18] local.ERROR: Class "app\Settings\TGBotSettings" not found {"exception":"[object] (Error(code: 0): Class \"app\\Settings\\TGBotSettings\" not found at /Users/enshtein/htdocs/mysite/vendor/spatie/laravel-settings/src/SettingsContainer.php:29)**

Before the update, according to the instructions, you could access the settings like this:
`
app(TGBotSettings::class)->no_answer`

and now it throws an error!

**Comment by rubenvanassche** [comment]
This should be fixed with settings v2.8.1
